### DISCUSSION TITLE
having control on available letters

### DISCUSSION THREAD

**Discussion by azinsharaf** [monkeytypegame/monkeytype#3911] [Questions] [answered]
hi,
Is there any way to choose the letters that words will be made of? I am a beginner and follow the KeyBR.com methodology which introduces letters one by one and gradually till you practice them well. It is similar to `pseudolang` funbox but it doesn't have all the letters. It would be nice to have a funbox that we can choose which letters we want to have/practice.

**Comment by Miodec** [comment, marked as answer]
The only way to do so at the moment is to either use your own training text in custom mode, or (also in custom mode) words filter, which can include and exclude words based on their letters.

**Reply by azinsharaf** [reply]
this custom mode is exactly what i needed. I can exclude letters and gradually add them. Thanks for your suggestion.

![image](https://user-images.githubusercontent.com/10101468/212174099-e8df5662-dda5-4bf7-a47e-9efb20a71152.png)
